operator/src/main/java/org/keycloak/operator/v2alpha1/crds/KeycloakSpec.java

```diff
@@ -16,13 +16,28 @@
  */
 package org.keycloak.operator.v2alpha1.crds;
 
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
+import org.keycloak.operator.v2alpha1.crds.keycloakspec.Unsupported;
+
 import java.util.Map;
 
 public class KeycloakSpec {
 
     private int instances = 1;
     private String image;
     private Map<String, String> serverConfiguration;
+    @JsonPropertyDescription(
+            "In this section you can configure advanced features, not production-ready, and not supported settings.\n" +
+            "Use at your own risk and open an issue with your use-case if you don't find an alternative way.")
+    private Unsupported unsupported;
```

Comment:
almost:
> everytime an "unsupported" feature becomes supported we __might__ introduce a breaking change and we __will expose supported__ fields in the root of spec

Reply:
well, if it's supported means we are going to remove it from `unsupported`

What I see is that the definition of the CRD should follow "business needs" not "time changing concepts".... this unsupported seems like a bag to put and remove things

I would introduce funtionalities in the CRD and the documentation (something that doesnt break anything) defines each feature as supported or not supported ( that algo can mean different things between Keycloak upstream and RHSSO product )

Reply:
This implementation implements what discussed for the issue #9729

Reply:
@jonathanvila The `unsupported` gate is for features that we really don't plan to support ever. It is mean for features that we want to provide, if someone really needs it, but with no guarantees it will work properly and with limited support from our side.

But we can discuss different naming for this.